packages/babel-parser/test/fixtures/typescript/class/modifier-name-parameters/input.ts

```diff
@@ -0,0 +1,4 @@
+class Foo {
+  constructor(private, public, static) {
```

Comment:
This should be an error, because `private`, `public` and `static` are reserved words in strict mode.

Reply:
`babel-parser` doesn't check reserved word for TypeScript? https://github.com/babel/babel/blob/b564368d6efc58038468503ae73d006b0679f0a1/packages/babel-parser/src/plugins/typescript/index.js#L1962-L1972

Reply:
Oh right, I didn't remember about that.